library/src/main/java/com/chuckerteam/chucker/internal/ui/transaction/TransactionPayloadFragment.kt

```diff
@@ -48,6 +51,8 @@ internal class TransactionPayloadFragment :
 
     private lateinit var viewModel: TransactionViewModel
 
+    private val uiScope = CoroutineScope(Dispatchers.Main) + Job()
```

Comment:
You can use `import kotlinx.coroutines.MainScope` instead.

Reply:
👍